subsys/testsuite/include/tc_util.h

```diff
@@ -15,7 +15,13 @@
 #include <shell/shell.h>
 #include <sys/printk.h>
 
+#if defined CONFIG_ZTEST_TC_UTIL_USER_OVERRIDE
+#include "tc_util_user_override.h"
```

Comment:
Change to from " to <.

" includes should only be used when the include path is relative from the file being processed.